libraries/message/src/lib.rs

```diff
@@ -29,9 +33,13 @@ fn versions_compatible(
     crate_version: &semver::Version,
     specified_version: &semver::Version,
 ) -> Result<bool, String> {
-    let req = semver::VersionReq::parse(&crate_version.to_string()).map_err(|error| {
-        format!("failed to parse crate version `{crate_version}` as `VersionReq`: {error}")
-    })?;
+    let current_version = semver::Version::parse(env!("CARGO_PKG_VERSION"))
+        .map_err(|error| format!("failed to parse current version: {error}"))?;
```

Comment:
This should be how we fill the `crate_version` argument, so we can just it that.

Reply:
Got it thanks!